src/number-formatting/short-format.js

```diff
@@ -40,6 +40,11 @@ function shortFormat(num, options) {
         }
     }
 
+    if (digitSize >= 10) { // > 999,999,500

```

Comment:
Nitpic. In comment example we have 9 digit number. Is that correct?

Reply:
Yes, it should be 9 digit number only, so any 9 digit number after mentioned number is rounded off to billion

Reply:
I'm not sure the comments make sense..
surely

if (digitSize >= 10) { // >= 1,000,000,000